examples/next-js/pages/_document.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Document, { Head, Main, NextScript } from 'next/document';
+
+export default class MyDocument extends Document {
+  render() {
+    return (
+      <html lang="en">
+        <Head>
+          <script
+            dangerouslySetInnerHTML={{
```

Comment:
So, it looks like this will only log client-side errors, right? I've set up my project for both client and server, though the source maps have continued to be an issue for me. I can post my config once I (hopefully) get this all working in my project!

Reply:
Correct. I figured for a minimal example that will help other get started in the right direction I would just use the browser SDK. Otherwise setting up Rollbar via their npm package starts to really get into the weeds with implementation details of Rollbar. Ultimately for those using the npm SDK it'd be a similar setup from a Rollbar initialization perspective:

```js
import RollbarSdk from 'rollbar';

new RollbarSdk({
  enabled: ${process.env.NODE_ENV === 'production'},
  accessToken: `${process.env.ROLLBAR_ACCESS_TOKEN}`,
  captureUncaught: true,
  captureUnhandledRejections: true,
  payload: {
    environment: `${process.env.NODE_ENV || 'production'}`,
    client: {
      javascript: {
        source_map_enabled: true,
        code_version: `${process.env.NEXT_BUILD_ID}`,
        guess_uncaught_frames: true,
      },
    },
  },
})
```

Reply:
I agree that only cover browser side is good enough, to keep it simple as possible.